README.md

```diff
@@ -1 +1,102 @@
+CHT Upgrade Service
+========================
 
+The CHT Upgrade Service provides an interface between CHT-Core API and Docker to allow easy startup and one-click upgrades from the CHT-Core Admin UI. 
+
+# Requirements
+
+* Docker
+* docker-compose
+
+# Installation and usage
+
+Installation and usage is achieved entirely with docker-compose. 
+
+#### Download and save CHT-Core docker-compose yml file. 
+
+Todo steps??
+
+#### Download and save CHT upgrade service docker-compose yml file.
+
+Todo steps??
+
+#### Export the environment variables:
+
+| Name                 | Required | Description                                                                                                                                                                                                                                  |
+|----------------------|----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `CHT_COMPOSE_PATH`   | yes      | Absolute path to the folder where the CHT docker-compose file is saved.                                                                                                                                                                      | 
+| `COUCHDB_USER`       | yes      | CouchDb main admin account username.                                                                                                                                                                                                         |
+| `COUCHDB_PASSWORD`   | yes      | CouchDb main admin account password.                                                                                                                                                                                                         | 
+| `COUCHDB_SECRET`     | yes      | Secret that is used by the CouchDB peers to communicate with each other and to generate authentication cookies. This field is mandatory if you are running CouchDb clustered mode to make the same authentication cookie valid on all nodes. |  
```

Comment:
This changed from "no" to "yes" in this edit, but the comment says it's only mandatory if clustered. Was that change intentional?

Reply:
I'm of two minds about this one. Right now, it gets a default value. What do you think?

Reply:
It's fine. The comment explains it clearly enough. And it's better that people provide it when not needed, rather than ignore it when it is needed, so erring to "yes" is safer. Thanks!